private.tf

```diff
@@ -10,8 +10,16 @@ resource "aws_subnet" "private" {
 
   vpc_id            = "${data.aws_vpc.default.id}"
   availability_zone = "${element(var.availability_zones, count.index)}"
-  cidr_block        = "${cidrsubnet(data.aws_vpc.default.cidr_block, length(var.availability_zones), length(var.availability_zones)  + count.index)}"
-  tags              = "${module.private_label.tags}"
+
+  cidr_block = "${
+    cidrsubnet(
```

Comment:
Next time, use a `null_resource` to define a trigger value for this complex calculation that is reused once again in `public.tf`

https://github.com/cloudposse/tf_label/blob/master/main.tf#L3